### PR TITLE
Bumping ember-cli-babel

### PR DESCRIPTION
**Pull request by arthirm**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "upfront"
   ],
   "dependencies": {
-    "ember-cli-babel": "6.8.1"
+    "ember-cli-babel": "6.17.0"
```
Comment (rwjblue) [inline package.json]: Can you add a `^` here please?

**Review by rwjblue — CHANGES_REQUESTED** [review]

**Comment by arthirm** [comment]
@rwjblue Updated please check

**Review by rwjblue — APPROVED** [review]

**Comment by arthirm** [comment]
@rwjblue Can you please release a new version?

**Comment by arthirm** [comment]
@nickiaconis Can you please release a new version for this?

**Pull request merged**